__tests__/app.test.js

```diff
@@ -276,3 +276,27 @@ describe("DELETE api/comments/:comment_id", () => {
       });
   });
 });
+describe("GET /api/users", () => {
+  test("should respond with 200 status code and an array of user objects with properties: username, name, avatar_url", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.users).toBeInstanceOf(Array);
+
+        response.body.users.forEach((user) => {
+          expect(user).toHaveProperty("username");
+          expect(user).toHaveProperty("name");
+          expect(user).toHaveProperty("avatar_url");
+        });
```

Comment:
How many users are you expecting to be returned? Make sure to check the length before asserting inside your forEach 👀